eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -49,40 +49,82 @@ param(
   [string]$PRBody = $PRTitle,
 
   [Parameter(Mandatory = $false)]
-  [string]$PRLabels
+  [string]$PRLabels,
+
+  [Parameter(Mandatory = $false)]
+  [string]$UserReviewers,
+
+  [Parameter(Mandatory = $false)]
+  [string]$TeamReviewers,
+
+  [Parameter(Mandatory = $false)]
+  [string]$Assignees
 )
 
 $headers = @{
   Authorization = "bearer $AuthToken"
 }
-
-$query = "state=open&head=${PROwner}:${PRBranch}&base=${BaseBranch}"
-
-function AddLabels([int] $prNumber, [string] $prLabelString)
+$baseURI = "https://api.github.com/repos"
+function SplitMembers ($membersString)
 {
-  # Adding labels to the pr.
-  if (-not $prLabelString) {
-    Write-Verbose "There are no labels added to the PR."
-    return
-  }
+  return @($membersString.Split(",") | % { $_.Trim() } | ? { return $_ })
+}
 
-  # Parse the labels from string to array
-  $prLabelArray = @($prLabelString.Split(",") | % { $_.Trim() } | ? { return $_ })
-  $prLabelUri = "https://api.github.com/repos/$RepoOwner/$RepoName/issues/$prNumber"
-  $labelRequestData = @{
-    labels = $prLabelArray
+function InvokeGitHubAPI($apiURI, $memberName, $additionSet, $body) {
+  $headers = @{
+    Authorization = "bearer $AuthToken"
   }
+  $errorOccurred = $false
+
   try {
-    $resp = Invoke-RestMethod -Method PATCH -Headers $headers $prLabelUri -Body ($labelRequestData | ConvertTo-Json)
+    if (-not $body)
+    {
+      $body = @{}
+      $body[$memberName] = @($additionSet)
+    }
+
+    $resp = Invoke-RestMethod -Method 'Post' -Headers $headers -Body ($body | ConvertTo-Json) -Uri $apiURI -MaximumRetryCount 3
+    Write-Host -f green "Members(s) [$body] added to: https://github.com/$RepoOwner/$RepoName/issue/$prNumber"
+    $resp | Write-Verbose
   }
   catch {
```

Comment:
Lets remove this try catch block and let it flow out instead of returning a boolean.